tests/postman/ElectionGuard API Tests.postman_collection.json

```diff
@@ -1095,59 +1411,5 @@
 				]
 			}
 		}
-	],
-	"variable": [
```

Comment:
I converted all the variables from collection variables to environment variables.  I also prefixed them all with `e2e-` so they could be easily differentiated from variables like urls that are used in the Web API collection.